code/game/g_client.c

```diff
@@ -1138,36 +1138,82 @@ void ClientUserinfoChanged( int clientNum )
     // Enforce the identities
     oldidentity = client->pers.identity;
 
-    if( level.gametypeData->teams )
-    {
-        s = Info_ValueForKey ( userinfo, "team_identity" );
+    if (!(client->sess.changeCount == 10 && client->sess.idLimit > level.time) ||
```

Comment:
Not a fan of the entire if clause to be honest - if (!(....)), please instead rewrite it so that it would be easy to track, if you negate the entire clause, then rewrite the clause so that you don't have to negate it in the end.